packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -892,7 +892,7 @@ class ReactDOMServerRenderer {
 
       let info = '';
       if (__DEV__) {
-        const owner = elementType._owner;
+        const owner = elementType && elementType._owner;
```

Comment:
Hmm wait. I don't see how this could work. Owner is defined on the `element`. Not on the `elementType`.

Reply:
In other words you want `nextElement._owner` here. We know `nextElement` exists because we already read `type` from it.